conans/client/cmd/build.py

```diff
@@ -11,7 +11,8 @@
 
 def cmd_build(app, conanfile_path, base_path, source_folder, build_folder, package_folder,
               install_folder, test=False, should_configure=True, should_build=True,
-              should_install=True, should_test=True):
+              should_install=True, should_test=True, layout_source_folder=None,
+              layout_build_folder=None):
```

Comment:
`layout_out_folder`?

Reply:
This would be confusing, as it comes from the ``conan build --build-folder`` argument, that we cannot change now without breaking.